runtime/src/main/java/com/datastax/oss/quarkus/runtime/internal/quarkus/CassandraClientStarter.java

```diff
@@ -36,10 +36,14 @@
 
   @SuppressWarnings("unused")
   public void startup(
-      @Observes StartupEvent event, QuarkusCqlSession session, CassandraClientConfig config) {
+      @Observes StartupEvent event,
+      QuarkusCqlSession sessionProxy,
+      CassandraClientConfig config,
+      QuarkusCqlSessionState quarkusCqlSessionState) {
     if (config.cassandraClientInitConfig.eagerSessionInit) {
       LOG.debug("Triggering eager initialization of Quarkus session at startup");
-      session.getName();
+      sessionProxy.getName();
+      quarkusCqlSessionState.setInitialized();
```

Comment:
I think this is redundant, you already call it in `createCassandraClient` above.

Reply:
you are right, removed